cobbler/api.py

```diff
@@ -858,7 +858,7 @@ def import_tree(self, mirror_url, mirror_name, network_root=None, autoinstall_fi
                     self.log("Network root given to --available-as is missing a colon, please see the manpage example.")
                     return False
 
-        import_module = self.get_module_by_name("manage_import_signatures").get_import_manager(self._collection_mgr, logger)
+        import_module = self.get_module_by_name("managers.import_signatures").get_import_manager(self._collection_mgr, logger)
```

Comment:
Ahhh that one I wasn't able to find... Thanks for fixing! We have to get the import tests working again!